config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -731,3 +731,49 @@ periodics:
         limits:
           cpu: 1
           memory: "2Gi"
+- name: ci-kubernetes-e2e-gce-network-metric-measurement
+  tags:
+    - "perfDashPrefix: network-performance"
+    - "perfDashJobType: performance"
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-periodics: "true"
+    preset-e2e-scalability-periodics-master: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-network
+    testgrid-tab-name: metric-measurement
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20201111-a263fd7-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=60
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --cluster=
+      - --extract=ci/latest
+      - --gcp-node-image=gci
+      - --gcp-nodes=102
```

Comment:
>> IIUC, you create pods that request 1cpu. The default node from our preset has a bit less than 1cpu allocatable, so you need to set appropriate node size.
>> Also - you need to accomodate for cluster-level addons.

 
> @VivekThrivikraman-est VivekThrivikraman-est 5 days ago 
As discussed made it 102 for time being.

That only add addresses the "cluster-addons" overhead.

Given that each node has actually 1cpu-epsilon allocatable capacity, none of your pods will have place to schedule.

There are two options:
(a) either we need to use 2-core VMs (@jkaniuk - do we have quota for that in our projects - we're planning a run per day so that's not that expensive)
(b) reduce the requests of your pods to something smaller

(a) might be better assuming we have enough quota (you should then use e2-standard-2)

Reply:
Do we really need 1 cpu request here?
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml#L27
request could be set to 0.5 (or whatever is a safe value), limit not set
cpu will only be limited if other components on the node use their share

Reply:
We could move to bigger nodes, but if possible, I would prefer to have all things standarized. More test will be added in the future, probably also based on this job, and this will be additional toil - ensuring that projects have quotas high enough, that derivate, maybe bigger jobs fit in the projects, etc.

Reply:
The pods were updated to use 0.5cpu